src/pages/workspaces/WorkspacesList/WorkspacesListTabs.test.ts

```diff
@@ -0,0 +1,118 @@
+import { act, fireEvent, render, screen } from '@testing-library/react';
+import { h } from 'react-hyperscript-helpers';
+import { updateSearch, useRoute } from 'src/libs/nav';
+import { CatagorizedWorkspaces } from 'src/pages/workspaces/WorkspacesList/CatagorizedWorkspaces';
+import { WorkspacesListTabs } from 'src/pages/workspaces/WorkspacesList/WorkspacesListTabs';
+import { asMockedFn } from 'src/testing/test-utils';
+import { defaultAzureWorkspace, defaultGoogleWorkspace } from 'src/testing/workspace-fixtures';
+
+// the FlexTable uses react-virtualized's AutoSizer to size the table.
+// This makes the virtualized window large enough for all rows/columns to be rendered in tests.
+jest.mock('react-virtualized', () => ({
+  ...jest.requireActual('react-virtualized'),
+  AutoSizer: ({ children }) => children({ width: 1000, height: 1000 }),
+}));
+
+jest.mock('src/libs/nav', () => ({
+  ...jest.requireActual('src/libs/nav'),
+  getLink: jest.fn(),
+  useRoute: jest.fn().mockImplementation(() => ({ params: {}, query: {} })),
+  updateSearch: jest.fn(),
+}));
+
+describe('The WorkspacesListTabs component', () => {
+  it('should render the workspaces of the current tab', () => {
```

Comment:
please add //Arrange //Act and //Assert section comments on these tests to help readability and follow AAA